e2e/cypress/integration/team_settings/join_closed_team_with_not_allowed_email_spec.js

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @team_settings
+
+import {getRandomId} from '../../utils';
+import * as TIMEOUTS from '../../fixtures/timeouts';
+
+describe('Team Settings', () => {
+    const randomId = getRandomId();
+    let testTeam;
+
+    before(() => {
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+            cy.apiAdminLogin();
+            cy.visit(`/${testTeam.name}`);
```

Comment:
I think the declared `testTeam` can be removed since it's used only here where `team` can be used directly.
```suggestion
            cy.visit(`/${team.name}`);
```